wde.go

```diff
@@ -55,8 +55,17 @@ recommended that your main function look like the the code below.
 
 wde.Run() will return when wde.Stop() is called.
 
-For this to work, you must import one of the go.wde backends. For
-instance,
+For this to work, you must import one of the go.wde backends. To
+automatically import the correct backend, add this to your imports:
+
+        import _ "github.com/skelterjohn/go.wde/init"
+
+This will register a backend with go.wde, allowing you to call
+wde.Run(), wde.Stop() and wde.NewWindow() without referring to the
+backend explicitly.
+
+If you would like to disable a backend, you must import all
```

Comment:
I'm assuming your trouble is with the line: "If you would like to disable a backend, you must import all...".

When I said "import all", I meant create files with the appropriate names so that the build rules will import the correct one.  It could maybe be changed to "create import files for all".

The idea was if you did not want your project to support say...OSX for some reason, you could manually import the other 2 backends by creating files with the appropriate filters rather than importing init.  This would effectively "disable" being able to build for OSX.  Perhaps this is not worth including, but I thought you did a good job explaining how to manually import so I didn't want to delete it.

All I really want with this PR is to update the documentation to say you can do:

`import _ "github.com/skelterjohn/go.wde/init"`

and it will register the right backend for you.  I won't be offended if you reject this PR and add it to the documentation in a better way.  *sigh* Writing is hard :(  Sorry for the confusion.